Utils/template.nf

```diff
@@ -6,10 +6,11 @@ process Command {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, file(input_file)
-
+    path input_file
+     
```

Comment:
A better example would be something like this?
```
tuple sample_id, file(input_file)
file(second_input_file)
```

Reply:
Agreed but it appears that file is now replaced with path (see DSL2 docs)